src/main/java/de/retest/configuration/Configuration.java

```diff
@@ -22,13 +22,9 @@ static boolean isLoaded() {
 		return instance != null && instance.retestWorkspace != null;
 	}
 
-	public static void ensureLoaded() {
+	public static synchronized void ensureLoaded() {
```

Comment:
I would rather prefer using a special lock with synchronize(lock) since using synchronized in a method locks on the object's instance. As a consequence if a other method uses the another synchronize on a method, which is completely unrelated to this method, we would synchronize without needing to.